src/context/FeatureFlaggedClerk/FeatureFlaggedClerk.tsx

```diff
@@ -70,7 +70,11 @@ export function useFeatureFlaggedClerk() {
 export function FeatureFlaggedClerkProvider({ children }: PropsWithChildren) {
   if (useFeatureFlagEnabled("use-auth-owa")) {
     return (
-      <ClerkProvider>
+      <ClerkProvider
+        signInUrl="/sign-in"
+        signUpUrl="/sign-up"
+        afterSignOutUrl="/"
+      >
```

Comment:
Clerk suggest you use environment variables for this. For us those won't appear under Git since they're stored in a GCS bucket making them quite hidden. I've opted for props to make this detail more visible (and versionable)